src/Sarif.Multitool.Library/RewriteOptions.cs

```diff
@@ -16,5 +16,33 @@ public class RewriteOptions : SingleFileOptionsBase
             Default = false,
             HelpText = "Sort results in the final output file.")]
         public bool SortResults { get; set; }
+
+        [Option(
+            "normalize-for-github",
```

Comment:
>"normalize-for-github" [](http://example.com/codeflow?start=12&length=22)

Question: the `Convert` command has the same `normalize-for-github` parameter, can we use `Convert` command to achieve the same?

Reply:
Convert means 'convert from some other non-SARIF format to SARIF.' This operation is a SARIF->SARIF transformation, not a 'conversion'.

One thing, we could use 'normalize-for-ghas' instead as a more precise designation (though 'ghas' may be more opaque to users.

I'm fine with either choice.

Reply:
use 'normalize-for-ghas' instead as a more precise designation
 ---- sounds good, will change it.

Reply:
use 'normalize-for-ghas' instead as a more precise designation
---- done.